.eslintignore

```diff
@@ -1,5 +1,5 @@
 **/__testfixtures__/*
 coverage
-test
+test/binCases/errors/parse/index.js
```

Comment:
This is needed because eslint-disable can't work on some files like:

<img width="400" src="https://user-images.githubusercontent.com/5961873/41962230-1f4a75dc-7a12-11e8-8fd6-8811c5d4757b.png" />

```
2:1  error  Parsing error: Unexpected token }
```

Reply:
But that's a SyntaxError. It would bomb an error anyway

Reply:
Yeah. This is error checking test. :sweat_smile: 
https://github.com/webpack/webpack-cli/blob/cd890a84facf29b9f9dec7cb876d3b085f4dc9b6/test/binCases/errors/parse/stdin.js#L10-L13